pkg/cmd/util/clientcmd/factory.go

```diff
@@ -510,12 +510,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if t.Spec.Paused {
 				return true, nil
 			}
-			t.Spec.Paused = true
+			body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
@deads2k I think CalculatePatches is an overkill for update simple like this, wdyt?